blocks/editable/patterns.js

```diff
@@ -189,7 +189,7 @@ export default function( editor ) {
 			return;
 		}
 
-		const firstText = content[ 0 ];
+		const firstText = editor.getBody().textContent;
```

Comment:
Maybe worth investigating if this has any side effects. This is no longer `firstText`.

Reply:
Right, not sure why we were only targeting the first text, I tried some patterns, it's working but not sure about all the usecases.

Reply:
I'll have a look at this too.